libraries/sbg-rs/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![no_std]
+
+#![allow(dead_code)]
+#![allow(non_snake_case)]
+#![allow(non_camel_case_types)]
+#![allow(non_upper_case_globals)]
+
+mod bindings;
+
+pub mod sbg;
```

Comment:
Maybe add documentation here, giving a quick overview of the purpose of this library?